### ISSUE TITLE
Menubar - keyboard navigation enter doesn't work

### ISSUE THREAD

**Issue by GitJacek** [primefaces/primeng#17672]
### Describe the bug

There is an issue with keyboard navigation in menubar when using custom template for menu item. We are unable to trigger menu command or route navigation using enter or space key. Event is blocked somewhere even if we add (keydown.enter) handler inside custom template.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/hcdxezjf?file=src%2Fapp%2Fimports.ts,src%2Fapp%2Fmenubar-template-demo.ts,src%2Fapp%2Fmenubar-template-demo.html

### Environment

Win 11

### Angular version

19.0.1

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Go to https://stackblitz.com/edit/hcdxezjf?file=src%2Fapp%2Fimports.ts,src%2Fapp%2Fmenubar-template-demo.ts,src%2Fapp%2Fmenubar-template-demo.html
2. Focus on menubar.
3. Naviagate throught menu using left arrow, right arrow, down arrow.
4. Try to trigger command on any menu item using enter. 

You should see console log with label in console but nothing appears. But if you click on menu item with mouse, console log is triggered.

### Expected behavior

User should be able to trigger enter event or command of MenuItem using only keyboard.

**Comment by GitJacek** [comment]
Any chance it get some attention? This seems crucial for keyboard only users.

**Comment by kimondev** [comment]
Is there an update according to this issue? Or a workaround

**Comment by GitJacek** [comment]
I haven't found any workaround to add keyboard accessibility to the current component...

**Comment by jonasbeltoft** [comment]
Would love an update on this
